doc/rados/operations/health-checks.rst

```diff
@@ -775,68 +775,69 @@ Device health
 config option.
 
-This warning only applies to OSDs that are currently marked "in", so
-the expected response to this failure is to mark the device "out" so
-that data is migrated off of the device, and then to remove the
-hardware from the system.  Note that the marking out is normally done
-automatically if ``mgr/devicehealth/self_heal`` is enabled based on
-the ``mgr/devicehealth/mark_out_threshold``.
+Because this alert  applies only to OSDs that are currently marked ``in``, the
+expected response to this expected failure is (1) to mark the OSD ``out`` so
+that data is migrated off of the OSD, and then (2) to remove the hardware from
+the system. Note that this marking ``out`` is normally done automatically if
+``mgr/devicehealth/self_heal`` is enabled (as determined by
+``mgr/devicehealth/mark_out_threshold``).
 
-Device health can be checked with:
+To check device health, run the following command:
 
 .. prompt:: bash $
 
    ceph device info <device-id>
 
-Device life expectancy is set by a prediction model run by
-the mgr or an by external tool via the command:
+Device life expectancy is set either by a prediction model that the mgr runs or
+by an external tool that is activated by running the following command:
 
 .. prompt:: bash $
 
    ceph device set-life-expectancy <device-id> <from> <to>
 
-You can change the stored life expectancy manually, but that usually
-doesn't accomplish anything as whatever tool originally set it will
-probably set it again, and changing the stored value does not affect
-the actual health of the hardware device.
+You can change the stored life expectancy manually, but such a change usually
+doesn't accomplish anything. The reason for this is that whichever tool
+originally set the stored life expectancy will probably undo your change by
+setting it again, and a change to the stored value does not affect the actual
+health of the hardware device.
 
 DEVICE_HEALTH_IN_USE
 ____________________
 
-One or more devices is expected to fail soon and has been marked "out"
-of the cluster based on ``mgr/devicehealth/mark_out_threshold``, but it
-is still participating in one more PGs.  This may be because it was
-only recently marked "out" and data is still migrating, or because data
-cannot be migrated off for some reason (e.g., the cluster is nearly
-full, or the CRUSH hierarchy is such that there isn't another suitable
-OSD to migrate the data too).
+One or more devices (that is, OSDs) are expected to fail soon and have been
+marked ``out`` of the cluster (as controlled by
+``mgr/devicehealth/mark_out_threshold``), but they are still participating in
+one or more Placement Groups. This might be because the OSD(s) were marked
+``out`` only recently and data is still migrating, or because data cannot be
+migrated off of the OSD(s) for some reason (for example, the cluster is nearly
+full, or the CRUSH hierarchy is structured so that there isn't another suitable
+OSD to migrate the data to).
```

Comment:
tsk tsk.  To which to migrate teh_dataz.

Reply:
Ending a clause with a preposition might be something up with which you will not put, but in the case of phrasal verbs such as we have here, I use the Germanic rules governing separable-prefix verbs (which are the same as the rules governing phrasal verbs) in preference to this Strunk-emulating-Quintillian Latinate Procrustean-bed-type rule, and I will trust that this is the kind of thing that our readers can deal with.

Reply:
>deal with
I see what you did there.

Doesn't seem analogous to a modal verb like _abfahren_ but you have the degree.

Reply:
> deal with
> I see what you did there.

At no time did I think that it would escape your notice, and I was right.

"Ich fahre ab." is plain old present indicative. "Ich kann abfahren." employs a modal verb (here also in the present indicative), which means that (because the clause already has within it one inflected verb ("kann", from the infinitive "koennen")), the other verb remains in its infinitival form ("abfahren"). But the modals are the ones that express possibility and ability, necessity, preference, and something else I'm forgetting right now (in German , "koennen", "muessen", "sollen", "moegen" and "wollen" are the modal verbs, equivalent respectively to the English "can", "have to" (not must, because the negation of "must" is not the same thing as the negation of "muessen"), "should", "like", and "want" ).

Good God at the parentheses.

Reply:
However, you are right if you mean that the infinitive form of the phrasal verb "deal with" is "to deal with" and not "to withdeal".

Reply:
Well, depends on whether one considers the preposition part of the verb, but I’m not going to debate you, Jerry